nautobot/core/wsgi.py

```diff
@@ -10,4 +14,23 @@
 # pattern from `nautobot.core.runner` to read environment or config path for us.
 nautobot.setup()
 
+# Use try/except because we might not be running uWSGI. If `settings.WEBSERVER_WARMUP` is `True`,
+# will first call `get_internal_wsgi_application` which does not have `uwsgi` module loaded
+# already. Therefore, `settings.WEBSERVER_WARMUP` to `False` for this code to be loaded.
+try:
+    import uwsgidecorators
+
+    @uwsgidecorators.postfork
+    def fix_uwsgi():
+        import uwsgi
+
+        logging.getLogger("nautobot").info(
```

Comment:
Any reason not to use `logging.getLogger(__name__)` (i.e. "nautobot.core.wsgi") instead?